xenmgr/Vm/Pci.hs

```diff
@@ -449,7 +453,37 @@ initCache = PciCache <$> enumerateDevices
                  , TE.decodeUtf8 (deDeviceName e)]
       | otherwise = findName es id
 
-
+pciGetResources :: String -> IO ([(Word, Word, Word)])
```

Comment:
Could you comment this function with an example of what the input looks like that you're parsing from the resource file?  Would help make the function more readable for people looking at this down the road.